monai/utils/safe_eval.py

```
@@ -0,0 +1,76 @@
+    ast.Div,
+    ast.FloorDiv,
+    ast.Pow,
+    ast.Mod,
+    ast.USub,
+    ast.UAdd,
+)
+
+
+def safe_eval(
+    expr: str,
+    globals: Mapping[str, Any] | None = None,
```

Ruff (0.15.17)

[error] Function argument `globals` is shadowing a Python builtin

(A002)